assets/pages/monthly-contributions/monthlyContributions.jsx

```diff
@@ -35,7 +35,10 @@ pageStartup.start();
 
 // ----- Redux Store ----- //
 
-const store = createStore(reducer, applyMiddleware(thunkMiddleware));
+const store = createStore(reducer, {
+  intCmp: getQueryParameter('INTCMP', 'gdnwb_copts_bundles_landing_default'),
```

Comment:
This probably isn't the right default parameter, it's for the bundles landing page.